controllers/numaresourcesscheduler_controller.go

```diff
@@ -252,14 +252,20 @@ func configParamsFromSchedSpec(schedSpec nropv1.NUMAResourcesSchedulerSpec, cach
 
 	var foreignPodsDetect string
```

Comment:
please expand the commit message to explain succintly the "why" of this change. Feel free to take parts of the PR description and paste them here (or, equivalently, the PR description should build and expand on the content of the most important commit message(s) of the series, which in this case is this one).